src/main/java/com/fortify/plugin/jenkins/steps/remote/MSBuildProjectType.java

```diff
@@ -0,0 +1,56 @@
+package com.fortify.plugin.jenkins.steps.remote;
+
+import com.fortify.plugin.jenkins.steps.Validators;
+import hudson.Extension;
+import hudson.util.FormValidation;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+
+public class MSBuildProjectType extends RemoteAnalysisProjectType {
+
+    private String dotnetProject;
+    private String dotnetAddOptions;
+
+    @DataBoundConstructor
+    public MSBuildProjectType() {
+    }
+
+    public String getDotnetProject() {
```

Comment:
Question:
Only .NET project is supported? That's why 'DotnetProject' is used? MSBuild actually can build non .NET project such as VC++ project.

Reply:
We don't have any requirements for supporting VC++ projects yet. In this ticket we are supposed to add a support of a remote translation for .NET projects by adding msbuild build tool along with Maven and Gradle. 'DotnetProject' is used in this context too.

Reply:
Thank you for the answer. I understood.